### PR TITLE
rebrand [Rr]edis->[Vv]alkey

### PR DESCRIPTION
**Pull request by daniel-house**
@madolson @zuiderkwast @stockholmux please review

### CODE REVIEW

**Review by stockholmux — APPROVED** [review]
LGTM

File: docs/management/debugging.md [hunk 1]
```diff
@@ -35,82 +35,82 @@ GDB can be used in two ways:
...
+without the help of the user, and the user is free to restart Valkey in their
 production environment.
 
-## Compiling Redis without optimizations
+## Compiling Valkey without optimizations
 
-By default Redis is compiled with the `-O2` switch, this means that compiler
-optimizations are enabled. This makes the Redis executable faster, but at the
-same time it makes Redis (like any other program) harder to inspect using GDB.
+By default Valkey is compiled with the `-O2` switch, this means that compiler
```
Comment (zuiderkwast) [inline docs/management/debugging.md:49]: (This is actually not true, but we can fix that another time.) It's good that we do all rebranding first.

File: docs/management/debugging.md [hunk 2]
```diff
@@ -134,13 +134,13 @@ simple as using the **bt** command:
 
     (gdb) bt
     #0  debugCommand (c=0x7ffc32005000) at debug.c:220
-    #1  0x000000010d246d63 in call (c=0x7ffc32005000) at redis.c:1163
-    #2  0x000000010d247290 in processCommand (c=0x7ffc32005000) at redis.c:1305
+    #1  0x000000010d246d63 in call (c=0x7ffc32005000) at valkey.c:1163
```
Comment (zuiderkwast) [inline docs/management/debugging.md:137]: There is no file called valkey.c. Also in redis there is no redis.c. This text must be very very old.... (We can fix it another time)

**Review by zuiderkwast — APPROVED** [review]
LGTM

**Pull request merged**